### ISSUE TITLE
Inadequate Color Contrast for Feedback (Send) Button

### ISSUE THREAD

**Issue by Hamza20203064** [swaraj-das/Collect-your-GamingTools#12]
The color contrast of the Feedback (Send) button is insufficient, making it difficult for users to read and interact with, particularly those with visual impairments or when viewing under different lighting conditions.
![Screenshot 2024-10-01 193220](https://github.com/user-attachments/assets/a34e2427-03d2-4b0b-b760-d1d071fbb9f3)
If you assigned this issue to me under gssoc-extd, I would love to work on that and add some new feachers.

**Comment by Hamza20203064** [comment]
Please assign this issue to me I will happy to work on that.

**Comment by Hamza20203064** [comment]
Sorry for the confusion. I am going to close both issues that I had raised.
